Training24/game.c

```diff
@@ -8,59 +8,136 @@
 // ------------------------------------------------------------------------------------------------
 
 #include <stdio.h>
+#include <string.h>
+#include <malloc.h>
+#include "game.h"
 
-/// <summary>Function to convert decimal to binary.</summary>
-void DecimalToBinary (int n) {
-   int binaryNum[32], i = 0, isNegative = n < 0;
-   if (isNegative) n = -n;                       // Work with positive equivalent
-   while (n > 0) {                               // Convert decimal to binary
-      binaryNum[i] = n % 2;
-      n /= 2;
-      i++;
+// Convert a decimal number to its binary representation

```

Comment:
use summary tagline for every functions **/// <summary> Converts a decimal number to binary/// </summary>
**